docs/Apps/Metabolomic Data/Polly-PeakML.md

```diff
@@ -0,0 +1,275 @@
+
+* Select Peak Detection flow - Targeted or Untargeted pipeline.
+
+    Polly-PeakML supports curation of peakgroups detected by both the available El-MAVEN's peak detection flow. 
+
+    ![Peak Detection Dialog](../../img/Polly-PeakML/peakDetectionFlow.png) <center>**Figure 5.** Select peak detection method</center>
+    
+* Select Polly-PeakML options
+
+    Move to the *Polly-PeakML* tab:  Click on the ‘peak curation’ checkbox. 
+
+    ![Peak Curation Dialog](../../img/Polly-PeakML/peakCurationDialog.png) <center>**Figure 6.** Select peak cuation</center>
+
+    Authenticate yourself for PeakML license: PeakML models are proprietary to the organization and require the user to be authorised for the same to use PeakML.
+
+    ![Authentication Dialog](../../img/Polly-PeakML/peakMLAuthorization.png) <center>**Figure 6.** Authenticate using Polly credentials</center>
+
+    Choose Models from the list: Different models can be available to different users. The list of the models depends on the user’s license for the same. 
+
+    ![peakML Models](../../img/Polly-PeakML/peakMLModels.png) <center>**Figure 7.** Select PeakML Model</center>
+
+    Slider - Using Slider, the user can set the range of the classified label to be bad or signal accordingly. For example, if the range is set to 0.0 to 0.3 (that is marked by red) this determines that the group has a probability of less than 0.3 would be noise. Similarly, the peak-groups with a probability of more than 0.6 (marked by green) would be a signal ( that is a good peak-group).
+
+    The gray region in slider from 0.3 to 0.6 is marked as - ‘Maybe good peak-group’
+
+    ![Slider](../../img/Polly-PeakML/slider.png) <center>**Figure 8.** Adjust slider</center>
+
+* Caveats 
+    
+    * The ML model is for feature detection. it can distinguish between good and noise peak-groups. It's not trained to distinguish between matched and unmatched peak-groups for targeted analysis. 
+
+    * For targeted analysis, it can eliminate noise peak-groups but required manual curation to select the matched peak-group from all the good peaks that are detected for a given m/z and RT.
+
+# Tutorial
+
+**Classified Peak Table**
+
+* Peak-tables have two additional columns than usual, namely, *‘Labels’* and *‘Classification probability’*. 
+
+* Labels - Polly-peakML classifies the peak-groups into six different classes.
+    
+    * Signal: Good peak-group.
+    * Signals with correlation: Good peak-group correlated with other peaks within the same RT window.
+    * Signal with cohort-variance: Good peak-group showing some interesting patterns across defined cohorts.
+    * Signals with both correlation and cohort-variance: Good peak-group with both the above-mentioned properties.
+    * Noise: Bad peak-group.
+    * MayBe Good: Peak-group may be a real signal or not.
+
+* Classification Probability - The probability value which we can assure how much model is certain that it is noise or signals.
```

Comment:
The probability value which defines the certainty of the model to classify the peak group as signal or noise.